plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -218,8 +270,13 @@ private function process_picture( OD_HTML_Tag_Processor $processor, OD_Tag_Visit
 				}
 
 				// Abort processing if a SOURCE lacks the required srcset attribute.
-				$srcset = $this->get_valid_src( $processor, 'srcset' );
-				if ( null === $srcset ) {
+				$srcset = $this->get_attribute_value( $processor, 'srcset' );
+				if ( ! is_string( $srcset ) ) {
+					return false;
+				}
+
+				// Abort if the srcset is a data: URL since there is nothing to optimize.
```

Comment:
Well, there could be _something_ to optimize. We can't preload the URL, but we could still add `fetchpriority=high` to the `IMG`.